### ISSUE TITLE
Color contrast issue on main navigation text

### ISSUE THREAD

**Issue by kara-franco** [chakra-ui/chakra-ui#22]
On [chakra-ui.com/getting-started](https://chakra-ui.com/getting-started), the Getting Started text does not have [proper color contrast ratio](https://contrast-ratio.com/#%23319795-on-white) for the font size (either the font needs to be increased to 14pt or the color changed). 

The Components text also fails [proper color contrast for any font size](https://contrast-ratio.com/#%23A0AEC0-on-white).

**Comment by segunadebayo** [comment]
Thanks for catching this @kara-franco . I guess making the text bold is also an option, correct?

**Comment by segunadebayo** [comment]
I've resolved this and will update the docs.
